code/processes/wdb.q

```diff
@@ -162,7 +162,7 @@ savetablesbypart:{[dir;pt;forcesave;tablename]
 		/- get list of distinct combiniations for partition directories
 		extrapartitions:.merge.getextrapartitions[tablename;extrapartitiontype];
 		/- enumerate data to be upserted
-		enumdata:.Q.en[hdbsettings[`hdbdir];value tablename];
+		enumdata:.Q.en[hdbsettings[`hdbdir];r:0!.save.manipulate[tablename;`. tablename]];
```

Comment:
It actually isn't needed here, thought it might be needed since it is referenced again in `savetables` in default mode but it isn't referenced again in `savetablesbypart` (which is the savetables function called in partbyattr mode) so should be able to get rid of that assignment.